homeassistant/components/binary_sensor/sleepiq.py

```diff
@@ -0,0 +1,15 @@
+"""
```

Comment:
@Teagan42 yet 😅

This started as a single sensor platform, and I hadn't factored it out to have a binary_sensor for is_in_bed yet. I pushed that up earlier.